read-the-docs-2016-stats.rst

```diff
@@ -0,0 +1,141 @@
+.. post:: Jan 11, 2017
+   :tags: stats, year-in-review
+
+Read the Docs 2016 Stats
+========================
+
+Congrats,
+you made it through 2016!
+Read the Docs has been rolling along,
+and we've had another interesting year as well.
+
+For a quick summary:
+* Our level of contributions and commits has gone down a good bit this year.
+* Our traffic and user growth has been quite steady, which is particularly impressive at the scale we're operating at.
+* Our efforts at funding the project have also started to bear fruit, and 2017 is looking to be the year that Read the Docs will become a sustainable project.
+
+We will go into more detail below about each of these details.
+
+.. note:: 
+
+	You can always see our stats for the last `30 days`_. 
+
+	Our posts from 2013_, 2014_, and 2015_ are also available.
+
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+
+Community
+---------
+
+This year, we had:
+
+* `18 people`_ who committed code (-50% from last year)
+* `1032 commits`_ (-46%)
+* `461 issues`_ - 236 open, 225 closed (-42%)
+
+.. https://github.com/rtfd/readthedocs.org/graphs/contributors?from=2016-01-01&to=2016-12-31&type=c
+.. git rev-list --count --all --max-age=1451606400 --min-age=1483228800
+.. is:issue  created:2016-01-01..2017-01-01 
+
+Our community activity is much lower this year,
+which is a multi-year trend that we've been seeing.
+I believe this is in part because of a maturing project and code base,
+and also because we've struggled getting more contributors.
+The code base and project are reasonably hard to set up,
+and we haven't had enough time and focus on onboarding contributors into the project.
+
+I'm not sure what the best path forward here is.
+I've personally onboarded and tried to build many people into ongoing contributors,
+and have gotten discouraged with the lackluster results over the years.
+My hope is that we can eventually make it easier to contribute and start to get more folks helping.
+
+The other path forward is to start using some of our funding to build a paid team who helps with the project.
+This seems like the most likely option given the history state of contributions.
+It would be great if projects could exist under ongoing volunteer maintenance,
+but there is a `plethora of examples`_ in the open source ecosystem of this not being the case.
```

Comment:
Similarly, perhaps this should be framed as how we are solving the fact that we can dedicate many resources to building community for our service. Also, though it may be at odds with gaining contributions in a way, we do prefer the idea of people being paid for their time. Ultimately, I'd like to promote someone from our community for this role, but there isn't anyone there.